parcels/field.py

```diff
@@ -946,8 +969,28 @@ def advancetime(self, field_new, advanceForward):
             self.data = np.concatenate((field_new.data[:, :, :], self.data[:-1, :, :]), 0)
             self.time = self.grid.time
 
+    def computeTimeChunk(self, data, tindex):
```

Comment:
I don't entirely understand why `data` needs to be an input argument here. Isn't that just `self.data`?

The reason I'm asking is because I think it would be useful for users to be able to do e.g. `fieldset.U.computeTimeChunk(tindex=0)` is e.g. they want to plot that. Now I see that users can do `fieldset.U.fieldset.computeTimeChunk(tindex=0, dt=1)`, but it's a bit redundant to do `Field.FieldSet`?